### ISSUE TITLE
How to get the indices fpr rasterizing descriptors?

### ISSUE THREAD

**Issue by BostonLobster** [alievk/npbg#11]
I saw the descriptors are stored in PointTexture class as nn.Parameters, and they are rasterized to image through `index_select`. I am wondering how to get these indices?

**Comment by seva100** [comment]
@BostonLobster are you referring to this line?
https://github.com/alievk/npbg/blob/c0cf6f2224bda3d4a5007e05c05fbaf5b34cf256/npbg/models/texture.py#L61 

In this case, the `ind` array should be suitable.

https://github.com/alievk/npbg/blob/c0cf6f2224bda3d4a5007e05c05fbaf5b34cf256/npbg/models/texture.py#L55

**Comment by BostonLobster** [comment]
@seva100 Yes, that is the line I referred. I am confused about where did you get the `ind` array.

**Comment by booker-max** [comment]
hi, in my understanding, the ids is obtained from the rendering of the scene, ids = inputs[:, 0], some parameters of the point cloud are passed into the NNScene class as some parameters of the scene, and then rendered to get the inputs, The pixel value of this inputs[:,0] is a corresponding index value from the 3D point cloud to the 2D plane.
